frontend/src/organisms/BaseFeeChart.tsx

```diff
@@ -95,10 +102,12 @@ function LiveChart(props: BaseFeeChartProps) {
           break;
         case "basefee":
           chartData.baseFeeFormatted = parseFloat(utils.formatUnits(block.baseFee, 'gwei'))
+          chartData.priorityFeeFormatted = parseFloat(utils.formatUnits(block.priorityFee, 'gwei'))
           break;
         case "gas":
           chartData.gasUsedFormatted = block.gasUsed.toNumber()
-          chartData.gasTargetFormatted = block.gasTarget.toNumber()
+          //chartData.gasUsedFormatted = block.gasUsed.mul(BigNumber.from(100)).div(block.gasTarget).toNumber()
```

Comment:
nit: lets remove all the commands you added to this screen since we no longer need to show gas Target

Reply:
i left this in case you decided to switch to/from percent or actual target